wpilibc/src/main/native/include/frc/smartdashboard/SendableChooser.h

```diff
@@ -113,17 +107,45 @@ class SendableChooser : public SendableChooserBase {
     }
   }
 
+  /**
+   * Returns the name of the selected option.
+   *
+   * If there is none selected, it will return the default option's name. If
+   * there is none selected and no default, it will return an empty string.
+   *
+   * @return The name of the option selected
+   */
+  std::string_view GetSelectedName() const {
+    std::scoped_lock lock(m_mutex);
+    if (m_haveSelected) {
+      return m_selected;
+    } else {
+      return m_defaultChoice;
+    }
+  }
+
   /**
    * Bind a listener that's called when the selected value changes.
    * Only one listener can be bound. Calling this function will replace the
    * previous listener.
-   * @param listener The function to call that accepts the new value
+   * @param listener The function to call that accepts the new name and new
+   * value
    */
-  void OnChange(std::function<void(T)> listener) {
+  void OnChange(std::function<void(std::string_view, T)> listener) {
     std::scoped_lock lock(m_mutex);
     m_listener = listener;
   }
 
+  /**
+   * Bind a listener that's called when the selected value changes.
+   * Only one listener can be bound. Calling this function will replace the
+   * previous listener.
+   * @param listener The function to call that accepts the new value
+   */
+  void OnChange(std::function<void(T)> listener) {
+    OnChange([listener](std::string_view val, T choice) { listener(choice); });
```

Comment:
```suggestion
    OnChange([listener = std::move(listener)](std::string_view val, T choice) { listener(choice); });
```
I think we want to move the listener instead of copying, but I'm not completely sure.